### PR TITLE
create init files for COLP page

### PR DESCRIPTION
**Pull request by Oysters1874**
I just created the most basic files for the new COLP page so that we can all work on it. 
I am wondering what we want to display in the drop-down list for COLP's name. I feel like City Owned and Leased Properties might be a little too long and COLP DB may be not clear enough.

### CODE REVIEW

**Review by SashaWeinstein — COMMENTED** [review]
Ok sorry I gave bad directions. I forgot to think about [devDB PR 208](https://github.com/NYCPlanning/db-colp/pull/208) which will send all qaqc files to a nested `qaqc/` folder.  I think the way to proceed is to move the files in `main/latest` to match this convention manually, approve that PR, and then test this code.
Unless you think we should merge this force and address in a following PR.

**Comment by abrieff** [comment]
I kind of think its fine (you could even just comment out the line for the modified names for now since it might not always work, the point is just to get something out so development can start happening with less merge conflicts

**Comment by SashaWeinstein** [comment]
Ok word that sounds right, the best way to get this out of development is limit the user to just choose the main branch and then change the code when we open up to more branches

**Comment by Oysters1874** [comment]
Deleted the dev branch at this moment.

File: src/colp/helpers.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from typing import Dict
+
+BUCKET_NAME = "edm-publishing"
+
+
+def get_data(branch):
+    rv = {}
+    url = f"https://edm-publishing.nyc3.digitaloceanspaces.com/db-colp/{branch}/latest/output"
+
+    #rv["modified_names"] = csv_from_DO(f"{url}/ipis_modified_names.csv")
```
Comment (SashaWeinstein) [inline src/colp/helpers.py:11]: This can be un-commented as the code works now

File: src/colp/colp.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        COLP is a list of uses on city owned and leased properties that includes geographic information as well as the type of use, agency and other related information. \
+        The input data for COLP is the Integrated Property Information System (IPIS), a real estate database maintained by the Department of Citywide Administrative Services (DCAS).
+        
+        ### About QAQC
+
+        The QAQC page is designed to highlight key scenarios that can indicate potential data issues in a COLP Database build.
+        """
+    )
+
+    data = get_data(branch)
```
Comment (SashaWeinstein) [inline src/colp/colp.py:24]: Black should automatically add a new line here. You can turn on black in settings under 
> Python > Formatting: Provider

**Review by SashaWeinstein — COMMENTED** [review]
Code looks good to me! One observation and one issue with regards to style

**Review by mbh329 — APPROVED** [review]

**Pull request merged**